src/common/ui/style/style.css

```diff
@@ -357,6 +345,18 @@ li {
   right: 0;
   bottom: 0;
 }
+.h-screen {
```

Comment:
Looks like these 4 rules were moved by accident?

Reply:
I was trying to override `height` of `abs-full` but seems no longer needed, now reverted.